.github/workflows/playwright.yml

```diff
@@ -67,10 +67,15 @@ jobs:
         path: test-results/
         retention-days: 5
 
+    - name: Check results file existence
+      id: check_results
+      uses: andstor/file-existence-action@v1
```

Comment:
@kornys please pin third-party actions using the SHA:
https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions